app/views/pilots/index.html.haml

```diff
@@ -1,6 +1,6 @@
 = content_for :section, 'Pilot Roster'
 
-.container-fluid.pt-2.pb-4
+.container.pt-2.pb-4
   .row
     .col-lg-12
```

Comment:
Would we not be served better by changing this to a `.col-lg-4`? Changing `container-fluid` to `container` has implications for mobile styling and other strange page width problems.